.github/workflows/wheel-builder.yml

```diff
@@ -378,10 +410,13 @@ jobs:
               PY_LIMITED_API="--config-settings=build-args=--features=pyo3/abi3-${{ matrix.PYTHON.ABI_VERSION }}"
           fi
 
-          uv build --wheel --require-hashes --build-constraint=$BUILD_REQUIREMENTS_PATH cryptography*.tar.gz $PY_LIMITED_API -o wheelhouse/
+          echo $PY_LIMITED_API
+          ls cryptography*.tar.gz
+
+          uv build --python='${{ steps.setup-python.outputs.python-path }}' -v --wheel --require-hashes --build-constraint=$BUILD_REQUIREMENTS_PATH cryptography*.tar.gz $PY_LIMITED_API -o wheelhouse/
         shell: bash
 
-      - run: uv venv
+      - run: uv venv --python='${{ steps.setup-python.outputs.python-path }}'
```

Comment:
Hmm, why is this required?

Reply:
uv's solver sees the free-threaded build as experimental, so you need need to explicitly opt in to using it. See https://github.com/astral-sh/uv/issues/8480#issuecomment-2430355206.

Reply:
Is tehre an issue tracking when they'll treat it as stable? Can you add a comment pointing to that issue or (if there's no issue) just saying that the --python can be removed when free threading is stable?